### PR TITLE
fid

### PR DESCRIPTION
**Pull request by vict0rsch**
content adapted from https://github.com/mseitzer/pytorch-fid

### CODE REVIEW

File: omnigan/transforms.py [hunk 1]
```diff
@@ -117,12 +117,12 @@ def __call__(self, data):
 
 class Normalize:
     def __init__(self, opts):
-        if opts.data.normalization == "default":
```
Comment (vict0rsch) [inline omnigan/transforms.py:120]: @tianyu-z this formulation introduces a loophole if opts don't have `opts.data.normalization`.

In general be careful with not doing `if/elif` without `else` or you might expose the code to leaky logic with unhandled cases.

Here if the default behaviour you want is `((0.485, 0.456, 0.406), (0.229, 0.224, 0.225))` then it should be in the else statement as I have done here

File: omnigan/trainer.py [hunk 2]
```diff
@@ -101,6 +103,73 @@ def __init__(self, opts, comet_exp=None, verbose=0, device=None):
             self.exp = comet_exp
         self.domain_labels = {"s": 0, "r": 1}
 
+    @torch.no_grad()
+    def paint(self, image_batch, mask_batch=None, resolution="approx"):
```
Comment (vict0rsch) [inline omnigan/trainer.py:107]: @tianyu-z @51N84D @alexrey88 @melisandeteng please have a look at this and tell me if this makes sense to y'all
